webapp/test_results_handler.go

```diff
@@ -99,6 +101,13 @@ func parseTestRunUIFilter(r *http.Request) (filter testRunUIFilter, err error) {
 		return filter, err
 	}
 
+	// Default to complete (aligned), stable runs.
+	if testRunFilter.IsDefaultQuery() {
```

Comment:
IMHO overriding `testRunFilter` here is a bit confusing, especially because we have already a similar behaviour in the `ToQuery` method with a subtle difference that `ToQuery(true)` only sets `complete`, but does not set `label=stable`. 

I know `ToQuery` has a different purpose and isn't suitable here. What about hoisting the logic of setting "default" params into another method, e.g. `SetDefaultParamsIfEmpty`, and calling that member both here and in `ToQuery`?

Reply:
I, uh, took the general idea but called it `OrDefault()`... looks kinda fluent:

    filters.OrDefault().ToQuery().Encode()